decidim-core/app/views/layouts/decidim/_social_meta.html.erb

```diff
@@ -0,0 +1,11 @@
+<meta name="twitter:card" content="summary_large_image" />
+<meta name="twitter:site" content="@<%= content_for(:twitter_handler) %>" />
+<meta name="twitter:title" content="<%= content_for(:meta_title) %>" />
+<meta name="twitter:description" content="<%= content_for(:meta_description) %>" />
+<meta name="twitter:image" content="<%= content_for(:meta_image_url) %>" />
+
+<meta property="og:url" content="<%= content_for(:meta_url) %>" />
+<meta property="og:type" content="article" />
+<meta property="og:title" content="<%= content_for(:meta_title) %>" />
+<meta property="og:description" content="<%= content_for(:meta_description) %>" />
+<meta property="og:image" content="<%= content_for(:meta_image_url) %>" />
```

Comment:
I guess we shouldn't include the meta tags if twitter_handler is empty or content_for(:meta_url) ?